packages/cli/__tests__/zosfiles/__system__/delete/vsam/cli.files.delete.vsam.system.test.ts

```diff
@@ -128,6 +128,20 @@ describe("Delete VSAM Data Set", () => {
             expect(response.status).toBe(0);
             expect(response.stdout.toString()).toMatchSnapshot();
         });
+
+        it("should successfully delete a VSAM data set with --ignore-not-found flag", async () => {
+            // create vsam
+            const createResponse = runCliScript(__dirname + "/__scripts__/command/command_invoke_ams_define_statement.sh",
+                TEST_ENVIRONMENT, [dsname, volume]);
+            expect(createResponse.status).toBe(0);
+
+            // now delete
+            const deleteResponse = runCliScript(__dirname + "/__scripts__/command/command_delete_vsam_data_set.sh",
+                TEST_ENVIRONMENT, [dsname, "--for-sure", "--ignore-not-found"]);
```

Comment:
Same question as for USS above - should we attempt the deletion a 2nd time and make sure it still works with `--inf` flag?